Car_Collision_Visulaization/src/data_visualization.cpp

```diff
@@ -0,0 +1,49 @@
+
+
+#include "data_visualization.h"
+
+
+Visualizer :: Visualizer()
+{
+    length_ = 0;
+    width_  = 0;
+}
```

Comment:
Delete this one. And make the other one explicit  as we can not have a visualizer without a window size